cpp/src/toppra/solver/qpOASES-wrapper.cpp

```diff
@@ -63,13 +63,11 @@ bool qpOASESWrapper::solveStagewiseOptim(std::size_t i,
 
   if (i < N) {
     value_type delta = deltas()[i];
-    // TODO self._A[0] access 0-th row ?
     m_A.row(0) << -2 * delta, -1;
     m_hA[0] = - xNext[0];
     m_lA[0] = - m_boundary;
 
-    // TODO self._A[1] access 1-th row ?
-    m_A.row(1).setZero();
+    m_A.row(1) << 2 * delta, 1;
     m_hA[1] = xNext[1];
     m_lA[1] = -m_boundary;
   }
```

Comment:
Here, when `i == N`, then the 2 first rows of `A, lA, hA` are left uninitialized. This code is mostly a copy-paste of the Python code and the same issue (if it is an issue) appears in the Python code. Is this expected ?

Reply:
In the Python code, for i == N, the inequalities become trivially true. This is because A[:2, :] is zero and lA and hA are initialized to -INF and INF respectively.

Reply:
Alright, then I have to fix the C++ code.

Reply:
Should be fixed now.